ui/src/components/Configuration/ConfigurationTableWrapper.vue

```diff
@@ -0,0 +1,327 @@
+    }
+
+    try {
+      //Actually Update the Server
+      await putProvisionDService(updatedProvisionDData)
+      await populateProvisionD(store)
+
+      closeConfigurationDrawer()
+
+      updateToast({
+        basic: 'Success!',
+        detail: `${mods[0]} to requisition definition ${mods[1]} successful.`,
+        hasErrors: false
+      })
+    } catch (e) {
+      updateToast({
+        basic: 'Error!',
+        detail: `${mods[0]} to requisition definition ${mods[1]} not successful.`,
+        hasErrors: true
+      })
+    }
+
+  } else {
+    // Inform User of Errors.
+    selectedProvisionDItem.errors = validatedItem
+  }
+  setLoading(false)
+}
+
+/**
+ * The user has made their deletion chicken switch/double check selection.
+ * @param selection Did the user choose to delete or not?
+ */
+const doubleCheckSelected = async (selection: boolean) => {
+  // The user opted to delete the entry
+  if (selection) {
+    // Update the local state to remove the value.
+    const copiedList = [...provisionDList.value]
+    copiedList.splice(doubleCheck.index, 1)
+
+    // Get a copy of the existing state.
+    const updatedProvisionDData = store?.state?.configuration?.provisionDService
+
+    // Remove the entry from the existing state.
+    updatedProvisionDData['requisition-def'] = ConfigurationHelper.stripOriginalIndexes(copiedList)
+
+    try {
+      await putProvisionDService(updatedProvisionDData)
+      updateToast({
+        basic: 'Success!',
+        detail: 'Deletion of requisition definition was successful.',
+        hasErrors: false
+      })
+    } catch (e) {
+      updateToast({
+        basic: 'Failure!',
+        detail: 'Deletion of requisition definition was NOT successful.',
+        hasErrors: true
+      })
+    }
+    populateProvisionD(store)
+
+  }
+  doubleCheck.active = false
+  doubleCheck.index = -1
+}
+
+/**
+ * Called when the user updates the page.
+ * @param newPage New Page Number
+ */
+const setNewPage = (newPage: number) => {
+  currentPage.page = newPage
+}
+
+/**
+ * Should we open the advanced panel when opened?
+ * Don't do it if the user has previously explicitly hidden
+ * the drawer.
+ */
+const advanceActiveUpdate = (newVal: boolean) => {
+  advancedActive.active = newVal
+}
+
+</script>
+
+<style lang="scss" scoped>
+@import "@featherds/styles/themes/variables";
+@import "@featherds/styles/mixins/typography";
+@import "@featherds/styles/mixins/elevation";
+.title {
+  @include headline3();
+}
+.title-padding {
+  margin: 20px;
+}
+.margin-bottom {
+  margin-bottom: 20px;
+}
+.white-bg {
+  background-color: var(--feather-background);
+  border: 1px solid #ebedf0;
+  margin-top: 16px;
+  margin-bottom: 24px;
+  @include elevation(2);
+}
+
+.flex {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.buttonIcon {
+  font-size: 24px;
+}
+</style>
+
+<style lang="scss">
+@import "@featherds/styles/mixins/typography";
+.button-wrapper {
```

Comment:
I guess by not using `scoped` the intention here is to have this class apply to the child components, but without `scoped` it is adding this css globally, which could cause conflicts. I think we'll need to either duplicate this in the child components or else use `:deep`, eg.
```
:deep(.button-wrapper) {
 ...
}
```